ui/src/app/views/concept-table/component.tsx

```diff
@@ -27,12 +27,21 @@ const styles = reactStyles({
     marginLeft: 10,
     verticalAlign: 'middle',
     color: colors.blue[3]
+  },
+  highlighted: {
+    color: colors.green[2],
+    backgroundColor: colors.green[2].alpha(0.2),
+    backgroundOpacity: '0.2',
```

Comment:
question: do we need backgroundOpacity if its specified in backgroundColor?

Reply:
we don't, that's a meaningless css property that doesn't exist that I was trying out and need to remove!